UIforETW/DirectoryMonitor.cpp

```diff
@@ -62,21 +61,23 @@ DWORD DirectoryMonitor::DirectoryMonitorThread()
 			return 0;
 
 		default:
-			assert(0);
+			ATLASSERT(0);
 			break;
 		}
 	}
   // Unreachable.
 }
 
+_Pre_satisfies_(this->hThread_ == 0)

```

Comment:
> It's too bad these annotations are ugly

[Don't hate them because they're beautiful](https://www.osr.com/blog/2015/02/23/sal-annotations-dont-hate-im-beautiful/).